app/src/pages/admin/allUsersPage/allUsersPage.jsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
```

Comment:
Well, that's not necessary because we expose `React` using `webpack`

Reply:
Fixed